backend/docker-compose.dev.yml

```diff
@@ -0,0 +1,18 @@
+version: '3'
```

Comment:
I've got a glitch:
```
~/workspace/woogle/backend % docker-compose -f docker-compose.dev.yml build
ERROR: Version in "./docker-compose.dev.yml" is unsupported. You might be seeing this error because you're using the wrong Compose file version. Either specify a version of "2" (or "2.0") and place your service definitions under the `services` key, or omit the `version` key and place your service definitions at the root of the file to use version 1.
For more on the Compose file format versions, see https://docs.docker.com/compose/compose-file/
```

Lets try to leave with older docker-compose v.2 format